homeassistant/components/wiz/__init__.py

```diff
@@ -89,8 +90,19 @@ def _async_push_update(state: PilotParser) -> None:
 
     await bulb.start_push(_async_push_update)
     bulb.set_discovery_callback(lambda bulb: async_trigger_discovery(hass, [bulb]))
-    await coordinator.async_config_entry_first_refresh()
-
+    await coordinator.async_refresh()
```

Comment:
Any reason to this, instead of intercepting and rethrowing?

```py
    try:
      await coordinator.async_config_entry_first_refresh()
    except ConfigEntryNotReady as err:
       await bulb.async_close()
       raise err
 ```

Reply:
That works as well 👍